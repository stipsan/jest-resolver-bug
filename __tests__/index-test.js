import salute from '..'

it('should salute', () => {
  expect(salute).toMatchSnapshot()
})