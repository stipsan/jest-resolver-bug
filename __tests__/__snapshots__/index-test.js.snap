// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should salute 1`] = `"hip,hip,array!"`;
